lib/index.js

```diff
@@ -360,10 +326,51 @@ module.exports = async function (options) {
     });
 
     try {
-        await ag.loadAssets(options.inputUrls);
-        await ag.populate({
-            followRelations: relationsQuery,
-            concurrency: options.concurrency || 100
+        const queue = (await ag.loadAssets(options.inputUrls))
+            .map(asset => ({asset, incomingRelationDescription: '<initial>'}));
+
+        const processedAssets = new Set();
+        async function processAsset({asset, incomingRelationDescription}) {
+            if (!processedAssets.has(asset)) {
+                processedAssets.add(asset);
+                t.push(null, {
+                    ok: true,
+                    name: 'loading ' + asset.urlOrDescription
+                });
+                try {
+                    await asset.load();
+                } catch (err) {
+                    handleError(err);
+                    return;
+                }
+                for (const relation of asset.externalRelations) {
+                    if (!relation.crossorigin && !relationTypeExclusions.includes(relation.type)) {
+                        queue.push({asset: relation.to, incomingRelationDescription: relationDebugDescription(relation)});
+                    }
+                }
+
+                // Conserve memory by immediately unloading assets (except Html which needs to be loaded
+                // for the later #fragment checks to work):
+                if (asset.type !== 'Html') {
```

Comment:
I guess svg includes across files is also a possibility, albeit quite unlikely with the limited browser support from IE